src/context.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import * as Emitter from 'tiny-emitter';
+
+interface PortalProviderProps {}
+
+export interface PortalValue {
+  subscribeToPortal: Function;
+  unsubscribeFromPortal: Function;
+  setPortal: Function;
+  getPortal: Function;
+}
+
+const PortalContext = React.createContext<PortalValue | null>(null);
+
+export class PortalProvider extends React.PureComponent<PortalProviderProps> {
+  private emitter: Emitter.TinyEmitter | null = null;
```

Comment:
Any reason not to initialize this here? Is there a point of having `null`?

```suggestion
  private emitter: Emitter.TinyEmitter = new Emitter.TinyEmitter();
```

Reply:
a83f655